base9/Settings.cpp

```diff
@@ -521,11 +518,10 @@ template<typename T> T Settings::getOrRequest (Node & n, string && f)
     {
         string input = "";
         cout << "Please enter your desired setting for field: '" + f + "':\n";
-        getline(cin, input);
-        cout << "Field '" + f + "' set to: " << input << endl;
```

Comment:
This sounds like a reasonable suggestion to me. Eventually we're going to want a way to cleanly output the used settings to the result files, but that's going to take some additional code to skip it when reading a result file. My hope is that we can unify the result reading code, but we'll see if that's feasible.

I'm fine with separating PRs by feature if that's how you want to present them. The only difference for me is how long we'll be reviewing one PR.